src/main/kotlin/org/rust/lang/refactoring/extractFunction/RsExtractFunctionDialog.kt

```diff
@@ -0,0 +1,64 @@
+package org.rust.lang.refactoring.extractFunction
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.refactoring.ui.NameSuggestionsField
+import org.rust.lang.core.psi.RsFunction
+import java.awt.Dimension
+import java.awt.GridLayout
+import javax.swing.JComboBox
+import javax.swing.JComponent
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+class RsExtractFunctionDialog(
```

Comment:
Take a look at `dialog` function: https://github.com/JetBrains/intellij-community/blob/master/platform/platform-impl/src/com/intellij/ui/components/components.kt#L135

Perhaps it'd be more convenient.